tests/tests_geomstats/test_fisher_rao_metric.py

```diff
@@ -20,7 +20,10 @@ def test_inner_product_matrix_shape(
         metric = self.Metric(information_manifold=information_manifold, support=support)
         dim = metric.dim
         result = metric.metric_matrix(base_point=base_point)
-        self.assertAllClose(gs.shape(result), (dim, dim))
+        if base_point.ndim == 1:
+            self.assertAllClose(gs.shape(result), (dim, dim))
+        else:
+            self.assertAllClose(gs.shape(result), (base_point.shape[0], dim, dim))
 
     def test_inner_product_matrix_and_its_inverse(
```

Comment:
I don't really understand the point of this test. It looks like it's just testing the gs.linalg.inv function. Should it maybe be removed ? @luisfpereira do you know what it's for ?

Reply:
probably to check it is full-rank @alebrigant. But in fact you are right, it tests mostly `gs.linalg.inv`. Checking `det` would be enough I guess.

Reply:
Ah yes you're probably right ! Then yes checking `det` would be more clear I think, if that doesn't take too long to change @tramy1258. Otherwise never mind.

Reply:
Yes I will change it briefly!